src/types/track.ts

```diff
@@ -13,6 +13,11 @@ export type NFTTrackJoin = {
   processedTrackId: string;
 }
 
+export enum SourceIPFS {
+  AUDIO = 'Audio',
+  ARTWORK = 'Artwork'
+}
```

Comment:
maybe just keep this within the processor/trigger since it's kinda specific just to those?